3주차/FE/sqaurelu/src/api/axios.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+import CovidState from '../models/CovidState';
+
+export const fetchCovidState = async (date) => {
+    const API_SERVICE_KEY = process.env.REACT_APP_API_SERVICE_KEY;
+
+    let queryParams = '?' + encodeURIComponent('ServiceKey') + '=' + API_SERVICE_KEY; /*Service Key*/
+    queryParams += '&' + encodeURIComponent('pageNo') + '=' + encodeURIComponent('1'); /**/
+    queryParams += '&' + encodeURIComponent('numOfRows') + '=' + encodeURIComponent('10'); /**/
+    queryParams += '&' + encodeURIComponent('startCreateDt') + '=' + encodeURIComponent(date); /**/
+    queryParams += '&' + encodeURIComponent('endCreateDt') + '=' + encodeURIComponent(date); /**/
+
+    try {
+        const res = await axios.get(`/openapi/service/rest/Covid19/getCovid19SidoInfStateJson${queryParams}`);
+        const { item } = res.data.response.body.items;
+        
+        const cityState = [];
+        const totalState = CovidState.make(item[18]); // 총계
+        for (let i = 1; i < 17; i++) { // 날짜의 가장 최신 데이터
+            cityState.push(CovidState.make(item[i]));
+        }
```

Comment:
api를 직접 확인할 수 없어서 정확한 의도를 파악하진 못했지만 map 함수를 이용해보시는 방법도 있을 것 같아요!
```suggestion
        const cityState = item.map((data) => CovidState.make(data));
        cityState.pop();
        const totalState = CovidState.make(item[18]); // 총계
```

혹은 for 문을 사용할 때에는 아래와 같은 방법도 있습니다!
```suggestion
        const cityState = [];
        const totalState = CovidState.make(item[18]); // 총계
        for (index in item) { // 날짜의 가장 최신 데이터
            if (index !== 18)
                cityState.push(CovidState.make(item[index]));
        }
```